src/Components/Components/src/Environment/ComponentEnvironment.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Services;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Environment
+{
+    /// <summary>
+    /// Provides information an environment specific features about the environment the components are running on.
+    /// </summary>
+    public class ComponentEnvironment
+    {
+        /// <summary>
+        /// Name of associated with the <see cref="ComponentEnvironment"/> when components are being remotely rendered on the server.
```

Comment:
Typo: `Name of associated with the`?

Also it would be good not to specify `on the server`, because we'd also use this one in Electron.

Reply:
Same typo in other comments below.